src/helpers/date.js

```diff
@@ -19,6 +19,17 @@ export const formatISODate = (date) => {
   return date;
 };
 
+// getAge: calculates the age of a given JS Date object
+export const getAge = (birthDate) => {
```

Comment:
Instead of `getAge`,  can you rename this to `calculateAge`? It's not so much a "getter"; more of a function to "calculate" the age.

Reply:
Change made